devel/models.py

```diff
@@ -7,13 +7,55 @@
 from django.contrib.auth.models import User, Group
 from django_countries.fields import CountryField
 from django.core.validators import MinValueValidator, MaxValueValidator
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.deprecation import MiddlewareMixin
 
 from .fields import PGPKeyField
 from main.utils import make_choice, set_created_field
 
 from planet.models import Feed
 
 
+class AuthTokenBackend(object):
+    def authenticate(self, request, username=None, password=None):
+        if 'X-Archweb-Token' in request.headers:
```

Comment:
Wouldn't it make sense to use `Authorization: Bearer` as header, this is a quite widely used way for such API and would also allow a lot of HTTP clients to use implemented helper methods for authenticate a request with a token. Otherwise you'd need to pass around custom header strings like `X-Archweb-Token`.
What are your thoughts about this?

Reply:
Sounds fine to me, it shouldn't conflict with things I suppose.